kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ClustersResourceIntegrationTest.java

```diff
@@ -28,16 +29,18 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 public class ClustersResourceIntegrationTest extends ClusterTestHarness {
```

Comment:
is this test only meant for zk? if yes can you add a comment with context to that effect..

Reply:
there is one test in this class that is only zk, others can be kraft/zk